sources/subs/Cache.class.php

```diff
@@ -415,6 +415,10 @@ protected function _build_prefix()
 	{
 		global $boardurl;
 
+		if (!file_exists(CACHEDIR . '/index.php'))
+		{
+			touch(CACHEDIR . '/index.php');
```

Comment:
Reminder for later (because at the moment it doesn't work), instead of touch'ing it, it would be nicer to use ```secureDirectory``` that re-creates it "properly" along with the htaccess.